### PR TITLE
Make ExactDC default

### PR DESCRIPTION
**Pull request by nikivazou**
This can create errors when fancy types are defined (e.g., the worker and wrapper constructor do not match) in which case LH suggests using the `no-exact-data-con` flag.

### CODE REVIEW

File: docs/blog/2016-10-06-structural-induction.lhs [hunk 1]
```diff
@@ -108,7 +108,7 @@ Reflecting Lists into the Logic
 To talk about lists in the logic, we use the annotation
 
 \begin{code}
-{-@ LIQUID "--exact-data-cons" @-}
+{- LIQUID "--exact-data-cons" @-}
```
Comment (facundominguez) [inline docs/blog/2016-10-06-structural-induction.lhs:111]: Now that this is the default, perhaps it needs not be mentioned anymore in the post.

**Review by facundominguez — COMMENTED** [review]
It looks good to me.

Eventually, we should have a bit of an explanation of what `--no-exact-data-cons` is disabling in `docs/mkDocs/docs/options.md`, and when to use it.

**Comment by facundominguez** [comment]
@nikivazou, would you like finishing this PR?

**Comment by facundominguez** [comment]
Hello @nikivazou, do you have any context to share on the decision to close this one?

**Comment by nikivazou** [comment]
Yes, I closed it because I do not believe I will any time soon find time to fix the failing tests...

**Pull request closed without merging**